ethstore/cli/src/main.rs

```diff
@@ -239,8 +239,13 @@ fn execute<S, I>(command: I) -> Result<String, Error> where I: IntoIterator<Item
 			.collect();
 		Ok(format_accounts(&accounts))
 	} else if args.cmd_import {
-		let src = key_dir(&args.flag_src)?;
-		let dst = key_dir(&args.flag_dir)?;
+		let password = match &args.arg_password.as_ref() {
+			&"" => None,
+			_ => Some(load_password(&args.arg_password)?)
```

Comment:
I don't like that you wrap potential errors in Option, use `ok()` instead to convert the result to an `Option`